TLM/TLM/Manager/Impl/AdvancedParkingManager.cs

```diff
@@ -2818,7 +2817,7 @@ bool citizenDebug
                             netManager.m_segments.m_buffer[segmentId].Info
                                       .m_lanes[laneIndex].m_finalDirection))
                     {
-                        if (CustomPassengerCarAI.FindParkingSpaceRoadSide(
+                        if (Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection.FindParkingSpaceRoadSide(
```

Comment:
consider a const for `Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection`